api/app/clients/sms/twilio.py

```diff
@@ -0,0 +1,75 @@
+import random
+from monotonic import monotonic
+from app.clients.sms import SmsClient
+from twilio.rest import Client
+
+# https://www.twilio.com/docs/sms/api/message#sms-status-values
+# these are for inbound messages
+# 'receiving'
+# 'received'
+#
+# something super frustrating here. We define a 'sent' status in models.py, but
+# the text that shows in ui is "Sent internationally". It would make sense to
+# map the "sent" status here to "sent", but that ends up showing confusing
+# stuff on page.
+#
+# That's why we map sent -> sending.
+twilio_response_map = {
+    'accepted': 'created',
+    'queued': 'sending',
+    'sending': 'sending',
+    'sent': 'sending',
```

Comment:
@dagoof this is right to map Twilio's `sent` to Notify's `sending`. As you found out, Notify's `sent` is used for when you send a text message internationally. As far as I understand it, when a message is sent internationally, we can not actually get the status back from the provider "yes, it was delivered", so they leave it as `sent`. But yeah I wonder why the state was not originally named `sent_internationally`.